terralab/utils.py

```diff
@@ -48,6 +49,14 @@ def wrapper(*args: str, **kwargs: str) -> Any:
             LOGGER.error(add_blankline_before(formatted_message))
             LOGGER.error(add_blankline_before(SUPPORT_EMAIL_TEXT))
             exit(1)
+        except MaxRetryError:
+            LOGGER.error(
+                add_blankline_before(
```

Comment:
As part of this ticket I realized that https://broadworkbench.atlassian.net/browse/TSPS-374 was already completed (which Jose suspected during refinement) 🙂